### DISCUSSION TITLE
Does magit-forge detect git config includeIf directives?

### DISCUSSION THREAD

**Discussion by dotinspace** [magit/forge#721] [Support]
I use different git users in different repos, via `includeIf` directives, but magit-forge seems not to respect that, and instead asks me for user and password. I am also using `authinfo.gpg` for credentials. Any one else experiencing the same, or am I doing something obviously wrong?

**Comment by tarsius** [comment]
> am I doing something obviously wrong?

You are obviously doing something wrong. Whether what you do is obviously wrong, I cannot know without sufficient details. (I am not in a good mood. Sorry.)

**Reply by tarsius** [reply]
Well actually, if you have not read https://github.com/magit/forge/wiki/Tips-and-Tricks#using-git-url-rewriting-instead, then that is obviously wrong.

**Reply by dotinspace** [reply]
Not a problem lacking good mood. Good answer to a stupid question, anyway.

**Reply by dotinspace** [reply]
Thanks, I'll RTFM, my bad.

**Reply by tarsius** [reply]
The M is a bit sparse here, so that might not be enough. If it doesn't help, describe all the things you have done, and I'll try to get back to you (probably late) next week.

**Reply by dotinspace** [reply]
So sorry about the fuss, it was just a matter of typo in token. Of course it had to be something like that.

**Reply by tarsius** [reply]
It happens. Have a nice day!
